src/Parser/DisciplinePriest/Modules/Items/BlindAbsolutionFourSet.js

```diff
@@ -0,0 +1,59 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+const BUFF_EXPIRATION_BUFFER_MS = 150;
+const REGULAR_PENANCE_COOLDOWN_MS = 9000;
+
+class BlindAbsolutionFourSet extends Module {
+  _hasProccedInCombat = false; // Proccing before combat will likely be common
```

Comment:
Did you check if the `combatantinfo` event has this buff? If it does you may be able to make a cleaner solution